File: Medium/match_last_item.py
# https://edabit.com/challenge/oGkwLhmpys95rjtQ2
def match_last_item(lst):
    return "".join(str(item) for item in lst[:-1])==lst[-1]

print(match_last_item(['rsq', '6hi', 'g', 'rsq6hig']), True)
print(match_last_item([0, 1, 2, 3, 4, 5, '12345']), False)
print(match_last_item(['for', 'mi', 'da', 'bel', 'formidable']), False)
print(match_last_item([8, 'thunder', True, '8thunderTrue']), True)
print(match_last_item([ 1, 1, 1, '11' ]), False)
print(match_last_item(['tocto','G8G','xtohkgc','3V8','ctyghrs',100.88,'fyuo','Q','toctoG8Gxtohkgc3V8ctyghrs100.88fyuoQ']), True)

File: javascript/Medium/matchLastItem.js
// https://edabit.com/challenge/jbR9NupEL8zAZkbKx


function matchLastItem(arr) {
	return arr.slice(0,arr.length-1).join('')=== arr[arr.length-1]
}


console.log(matchLastItem(['rsq', '6hi', 'g', 'rsq6hig']), true)
console.log(matchLastItem([ 0, 1, 2, 3, 4, 5, '12345' ]), false)
console.log(matchLastItem([ 'for', 'mi', 'da', 'bel', 'formidable' ]), false)
console.log(matchLastItem([8, 'thunder', true, '8thundertrue']), true)
console.log(matchLastItem([ 1, 1, 1, '11' ]), false)
console.log(matchLastItem(['tocto','G8G','xtohkgc','3V8','ctyghrs',100.88,'fyuo','Q','toctoG8Gxtohkgc3V8ctyghrs100.88fyuoQ']), true)